### ISSUE TITLE
FP when reading email

### ISSUE THREAD

**Issue by hydrapolic** [EsadCetiner/roundcube-rule-exclusions-plugin#7]
Server: Apache 2.4.58
Modsecurity version: 2.9.7
CRS version: 4.0.0_rc2
Roundcube version: 1.6.5
Roundcube-rule-exclusions-plugin git commit: 5ae44d81ed9e25df6849aa58bc821d566e3d525e

Blocked when trying to read an email:

```
GET /roundcube/?_task=mail&_caps=pdf%3D1%2Cflash%3D0%2Ctiff%3D0%2Cwebp%3D1%2Cpgpmime%3D0&_uid=21422&_mbox=INBOX&_framed=1&_action=preview HTTP/1.1
```

```
Message: Warning. Pattern match "(?i)(?:supplied argument is not a valid |SQL syntax.*)MySQL|Column count doesn't match(?: value count at row)?|mysql_fetch_array\\(\\)|on MySQL result index|You have an error in your SQL syntax(?:;| near)|MyS(?:QL server version for the right syntax to ..." at RESPONSE_BODY. [file "/usr/share/modsecurity-crs/rules/RESPONSE-951-DATA-LEAKAGES-SQL.conf"] [line "307"] [id "951230"] [msg "mysql SQL Information Leakage"] [data "Matched Data: mysqlclient. found within RESPONSE_BODY"] [severity "CRITICAL"] [ver "OWASP_CRS/4.0.0-rc2"] [tag "application-multi"] [tag "language-multi"] [tag "platform-mysql"] [tag "attack-disclosure"] [tag "paranoia-level/1"] [tag "OWASP_CRS"] [tag "capec/1000/118/116/54"]
```

The email was sent by a developer, so it contained the keyword mysqlclient and matched:
sql-errors.data:MySqlClient.

For now I've excluded 951230 locally.

**Comment by hydrapolic** [comment]
Also hit the sibling rule 951220 with an email from a backup server with this message:

```
mysqldump: Error 2013: Lost connection to server during query when dumping table `tbl` at row: 1721028
```

```
ModSecurity: Warning. Pattern match "(?i)(?:System\\\\.Data\\\\.OleDb\\\\.OleDbException|\\\\[Microsoft\\\\]\\\\[ODBC SQL Server Driver\\\\]|\\\\[Macromedia\\\\]\\\\[SQLServer JDBC Driver\\\\]|\\\\[SqlException|System\\\\.Data\\\\.SqlClient\\\\.SqlException|Unclosed quotation mark after the character string|'80040e14' ..." at RESPONSE_BODY. [file "/usr/share/modsecurity-crs/rules/RESPONSE-951-DATA-LEAKAGES-SQL.conf"] [line "282"] [id "951220"] [msg "mssql SQL Information Leakage"] [data "Matched Data: SQL server during query when dumping table `tbl` at row: 163175<br>\\x0a</div></div></div>\\x0a\\x09\\x09\\x09</div>\\x0a\\x09\\x09</div>\\x0a\\x09</div>\\x0a</div>\\x0a\\x0a<!-- popup menus ->\\x0a<div id=\\x22attachmentmenu\\x22 class=\\x22popupmenu\\x22>\\x0a\\x09<h3 id=\\x22aria-label-attachmentmenu\\x22 class=\\x22voice\\x22>Attachment options</h3>\\x0a\\x09<ul class=\\x22menu listing\\x22 role=\\x22menu\\x22 aria-labelledby=\\x22aria-label-attachmentmenu\\x22>\\x0a\\x09\\x09<li role=\\x22menuitem\\x22><a id=\\x2..."] [severity "CRITICAL"] [ver "OWASP_CRS/4.0.0-rc2"] [tag "ap [hostname "mail.server.com"] [uri "/roundcube/index.php"] [unique_id "ZWKMJEU3EvUz8V8QzilKFQAAAAM"], referer: https://mail.server.com/roundcube/?_task=mail&_mbox=Tmp
```

**Comment by EsadCetiner** [comment]
@hydrapolic Thank you for your report, I've opened a pull request here to fix the issue: https://github.com/EsadCetiner/roundcube-rule-exclusions-plugin/pull/8

Please let me know if this fixes your issue

**Comment by hydrapolic** [comment]
> @hydrapolic Thank you for your report, I've opened a pull request here to fix the issue: #8
> 
> Please let me know if this fixes your issue

Yes, it did help, thank you!
